playerctl/playerctl-daemon.c

```diff
@@ -731,12 +789,13 @@ static void player_signal_proxy_callback(GDBusConnection *connection, const gcha
     if (is_properties_changed) {
         GVariant *interface = g_variant_get_child_value(parameters, 0);
         GVariant *properties = g_variant_get_child_value(parameters, 1);
-        player_update_properties(player, g_variant_get_string(interface, 0), properties);
+        is_properties_changed =
```

Comment:
Create a new variable for this. It's changed the meaning of the variable which can lead to subtle bugs.